fixtures/Store.yml

```diff
@@ -0,0 +1,7 @@
+entity: Store
+items:
+  store{1..100}:
+    brand: '@brand($current)'
+    position:
+      coordinates: ['{{address.latitude}}', '{{address.longitude}}']
```

Comment:
This will be hard to handle if we need to calculate itineraries in the future

Reply:
How shall we do so ?

Reply:
Create only a few stores with real locations, maybe ?